src/read_dta.cpp

```diff
@@ -437,10 +439,9 @@ List read_dta(FILE * file, const bool missing, const IntegerVector selectrows) {
   // skip into the data part
   fseeko64(file, rlength * nmin, SEEK_CUR);
 
-  for(uint32_t j=0; j<nn; ++j)
+  for(uint64_t j=0; j<nn; ++j)
```

Comment:
If I'm not mistaken, this is a bugfix to the partial reading. Stata format 118 allows size uint64_t and we were only reading uint32_t. Not sure if it was introduced during partial reading or if a bugfix to current master as well